game/templates/game/board.html

```
@@ -160,24 +235,65 @@
+    <span class="logo-white">CHECK</span><span class="logo-gold">ORA</span>
+</span>
             </a>
             <span class="badge" id="modeBadge">PVP</span>
         </div>
-        
+       
+
         <div class="auth-buttons">
+             
+            <button
+               id="theme-toggle"
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)